templates/openstackconfiggenerator/config/16.2/rendered-tripleo-config.yaml

```diff
@@ -28,6 +28,17 @@ resource_registry:
 {{- end }}
 {{- end }}
 
+# Parameter merge strategies
+parameter_merge_strategies:
+  ServiceNetMap: merge
+  VipSubnetMap: merge
+  EndpointMap: merge
+  ExtraConfig: merge
+{{- range $roleid, $role := .RolesMap }}
```

Comment:
roleid is unused, should it be _?

Reply:
same for all of the other loops in that file so I kept it consistent